includes/Admin/Dashboard.php

```diff
@@ -432,6 +432,7 @@ public function get_dashboard_settings(): array {
 			'locale'                  => $this->locale->get_locale_settings(),
 			'newStoryURL'             => $new_story_url,
 			'archiveURL'              => $this->story_post_type->get_archive_link(),
+			'archiveURLRaw'           => $this->story_post_type->get_archive_link( true ),
```

Comment:
`raw` implies this being an unsanitized/unescaped variable.

Let's rename this to something less ambiguous everywhere, e.g. `defaultArchiveURL`

Reply:
Another idea, was to break this up into object. So 

```
archiveURL: {
     current: '...',
     'default': '...',
}
```

Thoughts?

Reply:
I think two separate properties are good enough, we don't need extra nesting.

Reply:
Changed